### PR TITLE
Add release instructions and CI for helm releases

### PR DESCRIPTION
**Pull request by chainlink**

### CODE REVIEW

File: doc/release_instructions.md [hunk 1]
```diff
@@ -104,6 +112,15 @@ OR
...
  ```
 
+Note: When releasing an alpha/beta/RC version, ensure the helmchart is marked as a pre-release
+
+`etc/helm/pachyderm/Chart.yaml`
+
+```
+annotations:
+  artifacthub.io/prerelease: "true"
+```
```
Comment (nitinjainsj) [inline doc/release_instructions.md:122]: I am guessing this change needs to be pushed to the release branch. I'd add the `git commit -am ""` command and `git push`.

We want release instructions to be complete with each step called out.

**Review by nitinjainsj — APPROVED** [review]
One comment.

File: doc/release_instructions.md (hunk 1 above)
Comment (chainlink) [reply doc/release_instructions.md:122]: Sounds good, I'll move it to the section where we edit chart.yaml above and add the git instructions

Comment (chainlink) [reply doc/release_instructions.md]: @nitinjainsj  Done

**Pull request merged**